nginx-php-mysql/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: "3.8"
+services:
+  backend:
+    image: php:8.0.3-fpm-buster
+    secrets:
+      - db-password
+    depends_on: 
+      - db
+    volumes:
+      - ./backend:/var/www/html # this part seems repetitive (jb)
+  db:
+    image: mysql:8.0.19
```

Comment:
Can you use MariaDB instead of MySQL to be compliant with ARM64 architecture? Mysql doesn't provide ARM image ATM.